### ISSUE TITLE
show original right click context menu?

### ISSUE THREAD

**Issue by phith0n** [HC200ok/vue3-easy-data-table#299]
Hi, 

I found that the right click context menu has been prevent event if there is no `@contextmenu-row` listener.

How to configure if I want it show original context menu?

**Comment by jxn-30** [comment]
Since Version 1.5.44 this is possible by setting the `prevent-context-menu-row` to `false` (passing `:prevent-context-menu-row="false"`). See #290 for reference

**Comment by ricardobuquet** [comment]
I don't think hiding context menu is a good user experience.  If my mom wants to copy/paste text she will use context menu to pick copy.  Please don't make my mom call me to copy text, context menu should be visible by default.

**Comment by jxn-30** [comment]
I do not like the default behaviour of the context menu being hidden either.
Changing this would now be a breaking change, thus I did set the default value of the new prop to `false`.

Including a breaking change would require at least an increase in minor version in terms of developer experience.

**Comment by JshGrn** [comment]
Yeah this default behavior is pretty crazy, shame we cant set defaults when the component is setup. Even for my mum who wants to right click to copy and paste as above, as a dev, not being able to right click inspect on a custom button in a slot is FRUSTRATING!! :) 

Is there a plan to make the default false?
